spec/commands_spec.rb

```diff
@@ -208,45 +211,37 @@ def rakefile_source
     end
 
     it "should create a new gem prefixed with nesta-plugin" do
+      @command.stub(:specify_gem_dependency)
       @command.should_receive(:system).with('bundle', 'gem', @gem_name)
-      begin
-        @command.execute
-      rescue Errno::ENOENT
-        # This test is only concerned with running bundle gem; ENOENT
-        # errors are raised because we didn't create a real gem.
-      end
+      @command.execute
     end
 
     describe "after gem created" do
       def create_gem_file(*components)
         path = File.join(@plugins_path, @gem_name, *components)
-        FileUtils.makedirs(File.dirname(path))
         File.open(path, 'w') { |f| yield f if block_given? }
         path
       end
 
+      def lib_path(*parts)
+        File.join(@gem_name, 'lib', *parts)
+      end
+
       before(:each) do
-        @required_file = create_gem_file('lib', "#{@gem_name}.rb")
-        @init_file = create_gem_file('lib', @gem_name, 'init.rb')
         @gem_spec = create_gem_file("#{@gem_name}.gemspec") do |file|
           file.puts "  # specify any dependencies here; for example:"
           file.puts "end"
         end
-      end
-
-      after(:each) do
-        FileUtils.rm(@required_file)
-        FileUtils.rm(@init_file)
+        @command.execute
       end
 
       it "should create the ruby file loaded on require" do
-        @command.execute
-        File.read(@required_file).should include('Plugin.register(__FILE__)')
+        required_path = lib_path("#{@namespace_path}.rb")
+        File.read(required_path).should include(%Q{Plugin.register(@namespace_path)})
```

Comment:
Line is too long. [85/80]<br>Use `%` instead of `%Q`.<br>Do not use `%Q` unless interpolation is needed.  Use `%q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.